nvse/nvse/ScriptAnalyzer.cpp

```diff
@@ -473,19 +473,72 @@ std::string StringForNumericParam(ParamType typeID, int value)
 		{
 			return g_actorValueInfoArray[value]->infoName;
 		}
+	case kParamType_Axis:
+		{
+			return value == 88 ? "X" : value == 89 ? "Y" : value == 90 ? "Z" : "<unknown axis>";

```

Comment:
You can use the characters instead of the ascii codes for this like:
			return value == 'x' ? "X" : value == 'y' ? "Y" : value == 'z' ? "Z" : "<unknown axis>";